source/openapi3/split.yaml

```diff
@@ -2063,10 +2065,10 @@ paths:
             items:
               type: string
             enum:
-              - proposed
-              - accepted
-              - declined
-              - cancelled
+              - [proposed]
+              - [accepted]
+              - [declined]
+              - [cancelled]
```

Comment:
Am curious about the square brackets. Won't that expand to:
```yaml
              -
                - proposed
              -
                - accepted
              -
                - declined
              -
                - cancelled
```

Reply:
It does, the schema type is set as `array` [above](https://github.com/zeptofs/api-documentation/blob/ccebe3f33445626c352eda665e856a26d667b78f/source/openapi3/split.yaml#L2064), so the valid values should be arrays of string, unless the schema type of `array` is wrong for the `status` property?

Reply:
I believe the intention is to accept an array of strings, each of which must belong to the enum.